assets/src/components/Histogram.js

```diff
@@ -1,11 +1,17 @@
+import React, { useState } from 'react'
 import createHistogram from './d3/createHistogram'
-import withResponsiveness from './withResponsiveness'
-import createChartComponent from './createChartComponent'
-import compose from '../util/compose'
-
-const Histogram = compose(
-  withResponsiveness,
-  createChartComponent
-)(createHistogram)
+import useChart from '../hooks/useChart'
+import useResponsiveness from '../hooks/useResponsiveness'
+
+function Histogram (props) {
+  const [el, setEl] = useState(null)
```

Comment:
can this have better names for understanding purposes?

Reply:
The `[el, setEL]` bit? Yes, how about `[element, setElement]`. Or maybe `[domElement, setDomElement]`?

Reply:
[domElement, setDomElement]

Reply:
Resolved here: https://github.com/tl-its-umich-edu/my-learning-analytics/pull/499/commits/795a1c5b70445bff14a2aa344c47419084e9605b